core/domain/feedback_services.py

```diff
@@ -902,6 +902,7 @@ def _add_feedback_message_reference(user_id, reference):
         enqueue_feedback_message_batch_email_task(user_id)
 
 
+@transaction_services.transaction_wrapper
```

Comment:
'transaction_wrapper' isn't really clear. How are decorators usually named? I'd expect something like 'run_in_transaction' like the underlying method to be clearer. What do you think?